assignment.md

```diff
@@ -39,48 +39,40 @@ OVERALL QUESTIONS
 # Managing code via the command line
-Git offers you many ways to manage your code both locally and remotely. Create branches to separate updates from production code, create patches, test ideas, and merge your changes into the codebase. The command line makes it easy to download, review, modify, and push changes between your local machine and a remote repository. <!-- New audiences may require a more in-depth intro, so we're leaving room for that here. -->
-
-<!-- Thought again; we could either link off to docs around local/remote branches and repositories if users need a refresh on this documentation or if they're newer to git and would require this information. -->
+Git offers you many ways to manage your code both locally and remotely. Create branches to separate updates from production code, create patches, test ideas, and merge your changes into the codebase. The command line makes it easy to download, review, modify, and push changes between your local machine and a remote repository.<!-- New audiences may require a more in-depth intro, so we're leaving room for that here. -->
 
 ## Git code management commands
 
-<!-- EAJ NOTE: Rearranging because the foundations build on each other -->
-Git includes three commands that you can use to review, update, and publish your changes to and from your local and remote repositories.
-- `fetch` - Retrieve code updates from a remote repository to place into your tracking branch.
-- `pull` - This command retrieves changes from a remote branch to place into your tracking branch before merging them into a local branch.
-- `push` - Send changes from a local branch to a remote repository.
+Git includes three commands that you can use to review, retrieve, and publish changes to and from your local and remote repositories:
+- `git fetch` - Retrieve code updates from a remote repository to place into your tracking branch.
+- `git pull` - This command retrieves changes from a remote branch to place into your tracking branch before merging them into a local branch.
+- `git push` - Send changes from a local branch to a remote repository.
 
 Find more information about these commands in the sections to follow.
 
-### Retrieving code updates (`fetch`)
-The `fetch` command first checks your local branch to determine whether an associated tracking branch exists remotely. If the tracking branch exists, `fetch` retrieves the changes and history from the tracking branch and pulls those changes into the <!-- tracking? remote? --> branch.
+### Retrieving updates from a branch (`git fetch`)
+The `git fetch` command retrieves the changes and history from a tracking branch to a locally-cached version of your remote repository. The `git fetch` command gives you an opportunity to review the changes that others have made to the tracking branch before merging them with the assets in your local branch. You can use `git fetch` to resolve conflicts before they are merged to your local branch.
```

Comment:
This does a better job of providing a fuller view of what branches are updated and where, because there was a question around what "download" meant.